src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ESResearchPurposeDAO.scala

```diff
@@ -0,0 +1,11 @@
+package org.broadinstitute.dsde.firecloud.dataaccess
+
+import org.broadinstitute.dsde.firecloud.model.DataUse.ResearchPurpose
+import org.elasticsearch.index.query.BoolQueryBuilder
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class ESResearchPurposeDAO(ontologyDAO: OntologyDAO) extends ResearchPurposeDAO with ElasticSearchDAOResearchPurposeSupport {
```

Comment:
Maybe something like ESResearchPurposeSupport? But I'm fine with this, the name isn't all that important to me.

Reply:
I don't like either of those but can't think of anything better. Will go with `ResearchPurposeSupport` because it fits with existing names.

Reply:
👍 to `ResearchPurposeSupport` for consistency. For devs working in the other similarly named areas, this will make more sense than having to dig around for something else.